config/maximal-config-example.toml

```diff
@@ -33,6 +33,13 @@ port = 7300
 # that is used by the `indexer-agent`. It is expected that `indexer-agent` will create
 # the necessary tables.
 postgres_url = "postgres://postgres@postgres:5432/postgres"
```

Comment:
Hey, for some reason our software doesn't have an error by having both. Could you check here https://serde.rs/container-attrs.html#deny_unknown_fields?